tests/src/FunctionalJavascript/ThunderJavascriptTestBase.php

```diff
@@ -284,4 +284,23 @@ public function runCron() {
       ->click();
   }
 
+  /**
+   * Click article save option based on index of action.
```

Comment:
I think we should document options here -> it will help us in future when we use this method.

Some list:
1 - Save and continue
2 - Save as unpublished (default)
3 - Save and publish